src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/prisonapi/PrisonerApiServiceTest.kt

```diff
@@ -579,86 +500,109 @@ internal class PrisonerApiServiceTest {
    * - { indexSentence: 1, sentencesInSequence: {1=[2]} } Sentence with single consecutive
-   * - { indexSentence: 3, sentencesInSequence: {3=[4], 4=[5]} } Sentence with multiple consecutive
+   * - { indexSentence: 3, sentencesInSequence: {3=[4], 4=[5]} 5=[14]} Sentence with multiple consecutive, including one out of sequence
    * - { indexSentence: 6, sentencesInSequence: {6=[7, 8]} } Sentence with single concurrent consecutive set
-   * - { indexSentence: 9, sentencesInSequence: {9=[10, 11], 10=[12, 13]} } Sentence with multiple concurrent consecutive sets
+   * - { indexSentence: 9, sentencesInSequence: {9=[10, 11], 10=[12, 13]} } Sentence with multiple concurrent consecutive sets, 12 and 13 to be sorted by same end date but different courts
    */
   val sentencesForSequencesFirst = listOf(
     Sentence(
       bookingId = defaultBookingId,
       sentenceSequence = 0,
       consecutiveToSequence = null,
-      courtDescription = "First Booking Court",
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       bookingId = defaultBookingId,
       sentenceSequence = 1,
       consecutiveToSequence = null,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       bookingId = defaultBookingId,
       sentenceSequence = 2,
       consecutiveToSequence = 1,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
       sentenceSequence = 3,
       consecutiveToSequence = null,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
       sentenceSequence = 4,
       consecutiveToSequence = 3,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
       sentenceSequence = 5,
       consecutiveToSequence = 4,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
       sentenceSequence = 6,
       consecutiveToSequence = null,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
       sentenceSequence = 7,
       consecutiveToSequence = 6,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
       sentenceSequence = 8,
       consecutiveToSequence = 6,
+      courtDescription = defaultBookingCourt,
     ),
 
     Sentence(
       defaultBookingId,
       sentenceSequence = 9,
       consecutiveToSequence = null,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
       sentenceSequence = 10,
       consecutiveToSequence = 9,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
       sentenceSequence = 11,
       consecutiveToSequence = 9,
+      courtDescription = defaultBookingCourt,
     ),
     Sentence(
       defaultBookingId,
-      sentenceSequence = 12,
+      sentenceSequence = 13,
       consecutiveToSequence = 10,
+      sentenceEndDate = LocalDate.now(),
```

Comment:
Small point: I think it'd be best to define a variable storing `LocalDate.now()` and use that instead. There's a (small but non-null) chance a test runs close to midnight and we get an unexpected test result.